src/gha_runner/gh.py

```diff
@@ -277,21 +277,28 @@ def wait_for_runner(
 
         """
         max = time.time() + timeout
+        start_time = time.time()
```

Comment:
🐑 maybe remove max and use `elapsed > timeout` below if you're going to add a start_time?

Reply:
Good call, done!